src/components/TodoList/TodoList.tsx

```diff
@@ -1,224 +1,84 @@
-/* eslint-disable */
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import classnames from 'classnames';
+import { useAppSelector } from '../../utils/hooks';
+import { useDispatch } from 'react-redux';
+import { setCurrentTodo } from '../../features/currentTodo';
+import { getFilteredTodos } from '../../utils/filter';
 
 export const TodoList: React.FC = () => {
-  return (
-    <>
-      <p className="notification is-warning">
```

Comment:
You should show it if there are no filtered todos:

![image](https://github.com/user-attachments/assets/d14c6979-3081-4d83-a4bf-7f24781a0505)

Working version example:
![image](https://github.com/user-attachments/assets/911afbd3-369c-40a3-bcdd-f72b2c3eda3e)